test/plugin/test_out_google_cloud_grpc.rb

```diff
@@ -310,6 +320,9 @@ def _undefined
 
   # Set up grpc stubs to mock the external calls.
   def setup_logging_stubs(should_fail = false, code = 0, message = 'Ok')
+    # Save the mock host in an instance variable, so later on when creating the
+    # logging driver, we can refer to this host with exactly the same port.
+    @mock_host = generate_mock_host
```

Comment:
We could make it a bit less magic by yielding the host into the block, so we can pass it into `create_driver`, but that would require changing every use of `setup_logging_stubs`, and this is a temporary change, so up to you.
Let's add this as a separate commit, though, so we can revert just that one without reverting the whole PR.

Reply:
I thought about it, but decided not to make changes to every test for a temporary workaround like this.